gradebookng/tool/src/webapp/styles/gradebook-gbgrade-table.css

```diff
@@ -43,7 +43,7 @@
 #gradeTableWrapper .handsontableInputHolder .out-of {
```

Comment:
There's a lot of standardization going on in the colors used here. I'm a little unfamiliar with the GB markup/css, so if anyone has concerns, please raise them.